libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilotImpl.kt

```diff
@@ -397,7 +400,10 @@ internal class MapboxCopilotImpl(
 
     internal companion object {
 
-        internal val gson = Gson()
+        internal val gson = GsonBuilder()
+            .registerTypeAdapterFactory(DirectionsAdapterFactory.create())
+            .registerTypeAdapter(Point::class.java, PointAsCoordinatesTypeAdapter())
```

Comment:
I think it would be nice if mapbox-java exposed a single point for adapter, so that:
1. other users could use it in their projects with 1 extra line of code
2. if we add new adapters, we don't need to add it to the copilot too

Reply:
Great idea! Could you cut a ticket in `mapbox-java` when you have a chance? 🙏